lib/matplotlib/colorbar.py

```diff
@@ -1164,6 +1191,47 @@ def remove(self):
             # use_gridspec was True
             ax.set_subplotspec(subplotspec)
 
+    def minorticks_on(self):
+        """
+        Turns on the minor ticks on the colorbar without extruding
+        into the "extend regions".
+        """
+        try:
+            ax = self.ax
+        except AttributeError:
```

Comment:
Can that happen? The attribute is defined in `ColorbarBase.__init__`.

Reply:
You are right... removed the try except block